lib/tasks/sitemap.rake

```diff
@@ -8,7 +8,10 @@ namespace :sitemap do
 
     output_directory = File.join(PROJECT_ROOT, "public")
     sitemap = Sitemap.new(output_directory)
-    sitemap_index_path = sitemap.generate(search_server.content_indices)
+    search_config = SearchConfig.instance
+    sitemap.generate(
+      index_names: search_config.content_index_names + [search_config.govuk_index_name]
```

Comment:
I think it makes sense here to pass in `search_config` rather than the index names, because the generator actually has a dependency on the whole thing (since it also accesses the singleton object).